src/generators/genhl.ml

```diff
@@ -4003,7 +4003,7 @@ let add_types ctx types =
 				| _ ->
 					false
 			in
-			if not ctx.is_macro then List.iter (fun f -> ignore(loop c.cl_super f)) c.cl_overrides;
+			if not ctx.is_macro then List.iter (fun f -> if has_class_field_flag f CfOverride then ignore(loop c.cl_super f)) c.cl_ordered_fields;
```

Comment:
Shouldn't this also traverse `cl_ordered_statics`?

Reply:
No because overrides are never static.

Reply:
Ah, I'm always confused with "override"/"overload". These words are too similar for me in english )